app/pages/dashboard/controller.test.js

```diff
@@ -32,7 +33,9 @@ describe('dashboard controller', () => {
       await getDashboardContext({ orgId: 'org-id', orgName: 'org1', accessToken: 'access_token' });
 
       expect(contextCreator.getContext.mock.calls.length).toEqual(1);
-      expect(contextCreator.getContext).toHaveBeenCalledWith({ orgName: 'org1', completedOrders: [], incompletedOrders: [] });
+      expect(contextCreator.getContext).toHaveBeenCalledWith({
+        orgName: 'org1', completedOrders: [], incompletedOrders: [], userIsProxy: undefined,
```

Comment:
Can we mock `getIsUserProxy` to test that the expected value is passed through rather than `undefined`?